### PR TITLE
Add feature to add all events from a race week to Apple Calendar app

### PR DESCRIPTION
**Pull request by qeude**
## Description

I added a feature to add all events from a race week to the Apple Calendar application. It basically uses AppleScript to access the Mac application, retrieve calendars, and add events to the selected calendar.

## Screencast
Screen Recording 2022-04-23 at 00 54 45

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by sxn** [comment]
hi @qeude - nice changes! Hope you don't mind, I've sneaked a small change in as well 😅 

A couple of mentions:
1. Could you update the `CHANGELOG.md` file to reflect this change?
2. Trying to add the sessions to a calendar I'm not allowed to write to results in the extension crashing:

https://user-images.githubusercontent.com/1155589/165083569-0f5f4fe9-3946-4d7a-b477-e1681cddaea4.mov

Let me know once you've had a chance to fix the issue or if you need some help either here or on [Slack](https://raycastcommunity.slack.com/) 😸

File: extensions/f1-standings/src/views/AddToCalendar.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    LocalStorage.setItem("f1:calendar", calendar);
+    await runAppleScript(
+      'tell application "Calendar"\n' +
+        `tell calendar "${calendar}"\n` +
+        raceDates
+          .map((date) => {
+            const endDate = moment(date[1]).add(2, "hours");
+            return `make new event with properties { summary:"🏎 ${getFlag(race.Circuit.Location.country)} ${
+              date[0]
+            } - ${race.raceName}", start date:(date "${moment(date[1]).format(
```
Comment (sxn) [inline extensions/f1-standings/src/views/AddToCalendar.tsx:55]: is `moment` necessary here? The extension already has a [formatDate](https://github.com/raycast/extensions/blob/main/extensions/f1-standings/src/utils/formatDate.ts) function which takes into account the user's locale 🤔 can'd we use that instead?

Comment (qeude) [reply extensions/f1-standings/src/views/AddToCalendar.tsx:55]: I did use `moment` so I can format to the specific needed format for AppleScript. IMO `moment` makes it much smoother in that case. 🤷

**Comment by qeude** [comment]
I'm going to take a look at these feedbacks ASAP 🙌🏼

**Comment by qeude** [comment]
@sxn No worries about adding some changes, it's always ok to improve an extension together 😄 
I've made the changes you requested:
- Changed the `CHANGELOG.md`.
- Changed the used AppleScript so it doesn't list calendars that are not writable.
- Didn't change [`moment.js` for the reason explained here](https://github.com/raycast/extensions/pull/1470#discussion_r857611893). Even if I tend to be a "no-third" dependency guy, I feel that `moment.js` is pretty mature to be used and useful for readability here.

**Comment by sxn** [comment]
nice! 🎉

**Comment by csigritz** [comment]
thx guys...

**Pull request merged**